src/profile/providers/SearchFilter.tsx

```diff
@@ -36,43 +35,21 @@ const INITIAL_STATE: IState = {
 
 export const ProfileSearchContext = createContext(INITIAL_STATE);
 
-const parseRange = (rangeString: string): [number, number] | null => {
-  const match = rangeString.match(/^\[\d\,\d\]$/);
-  if (!match) {
-    return null;
-  }
-  const [n, m] = match.map((k) => Number(k));
-  return [n, m];
-};
-
```

Comment:
Is this parsing not necessary anymore?

Reply:
I think I moved these things out of query-parameters and just into local state. Found it unnecessarily complicated to have them as query params at this moment in time